runtime/src/lib.rs

```diff
@@ -637,24 +668,35 @@ impl_runtime_apis! {
 		}
 
 		fn get_set_free_balance_extrinsic(account_id: AccountId, free_balance: Balance) -> <Block as BlockT>::Extrinsic {
-			let timestamp = Timestamp::now();
 			UncheckedExtrinsic::new_unsigned(
-				pallet_balances::Call::<Runtime>::set_free_balance { who: sp_runtime::MultiAddress::Id(account_id), new_free: free_balance, magic_number: timestamp.into() }.into()
+				pallet_balances::Call::<Runtime>::force_set_balance { who: sp_runtime::MultiAddress::Id(account_id), new_free: free_balance }.into()
```

Comment:
`force_set_balance` ensure only root is calling; I added again previous `set_free_balance` call so we have to preserve `pallet_balances::Call::<Runtime>::set_free_balance`